Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM php:8.3-apache
+
+ENV APACHE_DOCUMENT_ROOT /var/mapbender/application/public
+
+RUN apt-get update && apt-get install -y \
+        libpq-dev \
+        git \
+        vim \
+        unzip \
+        openssl \
+        bzip2 \
+        sqlite3 \
+        curl \
+        libcurl4-gnutls-dev \
+        libgd-dev \
+        libicu-dev \
+        libzip-dev \
+        libbz2-dev \
+        libxml2-dev \
+        libsqlite3-dev \
+        libldap2-dev \
+        libonig-dev \
+        postgresql-client
+
+RUN docker-php-ext-install pdo_pgsql
+RUN docker-php-ext-install curl gd intl mbstring zip bz2 xml pdo_sqlite ldap
```

Comment:
the php extension `json` is miossing here as far as I can see, unless it's already included in the base `php8.3:apache` package?

Reply:
json is bundled since PHP 8.0.0